endo_api/settings_base/env.py

```
@@ -0,0 +1,169 @@
+        "SESSION_COOKIE_SECURE": get_bool("DJANGO_SESSION_COOKIE_SECURE", False) or get_bool("SESSION_COOKIE_SECURE", False),
+        "CSRF_COOKIE_SECURE": get_bool("DJANGO_CSRF_COOKIE_SECURE", False) or get_bool("CSRF_COOKIE_SECURE", False),
+    }
+
+
+def _db_options_from_env() -> Dict[str, Any]:
+    # JSON override takes precedence
+    options_raw = os.environ.get("DJANGO_DB_OPTIONS")
+    if options_raw:
+        try:
+            return json.loads(options_raw)
+        except Exception as exc:  # noqa: BLE001
```

Ruff (0.12.2)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)